lib/ch.ex

```diff
@@ -55,6 +55,7 @@ defmodule Ch do
           | {:command, Ch.Query.command()}
           | {:headers, [{String.t(), String.t()}]}
           | {:format, String.t()}
+          | {:types, [String.t()]}
```

Comment:
It can also be an atom (`i8`, `:string`, etc.) or a tuple (`{:array, ...}`, etc.). It might make sense to add something like `Ch.Types.t` or `Ch.RowBinary.type` and use it here.